src/components/productSection/Product.jsx

```
@@ -0,0 +1,126 @@
+import React from 'react';
+import product1 from './images/product1.png';
+import product2 from './images/product2.png';
+import product3 from './images/product3.png';
+import subImage from './images/sub-image.png';
+
+const products = [
```

ESLint

[error] 'products' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)